java/meterpreter/stdapi/src/main/java/com/metasploit/meterpreter/stdapi/stdapi_sys_process_execute.java

```diff
@@ -65,10 +65,65 @@ public int execute(Meterpreter meterpreter, TLVPacket request, TLVPacket respons
         return ERROR_SUCCESS;
     }
 
+    // On Windows, Java quote-escapes _some_ arguments (like those with spaces), but doesn't deal correctly with some
+    // edge cases; e.g. empty strings, strings that already have quotes.
+    protected String escapeArg(String arg) {
+        if (System.getProperty("os.name").toLowerCase().contains("windows")) {
+            if (arg == "") {
```

Comment:
Looks like we don't have a linter for the java meterpreter wired up; but iirc we'll want a null safe `.equals` check here instead of `==`

https://www.geeksforgeeks.org/difference-between-and-equals-method-in-java/

Reply:
Done.